README.md

```diff
@@ -2,18 +2,26 @@
 
-This package makes use of a [resource overlay](https://source.android.com/docs/core/architecture/rros) to replace the list of hardcoded webviews with one that also includes the Bromite WebView. I personally find this method more straightforward and elegant, as it does not require root access nor the tedious process of installing Magisk modules or patching the system framework itself manually — if anything breaks the package can simply be removed. Moreover, the WebView itself does not need to be installed as a system app and has no potential risk of breaking SafetyNet.
+This package makes use of a [resource overlay](https://source.android.com/docs/core/architecture/rros) to replace the list of hardcoded webviews with one that also includes the Bromite WebView. I personally find this method more straightforward and elegant, as it does not require the tedious process of patching the system framework itself manually — if anything breaks the package can simply be removed. Moreover, the WebView itself does not need to be installed as a system app and has no potential risk of breaking SafetyNet — e.g. you can install it directly from F-Droid.
 
 ![The WebView implementation settings with the Bromite SystemWebView Overlay installed](screenshot.png)
 
-Although this method should work on all Android versions that support Bromite and it's WebView, **currently testing has only been done on LineageOS 19.1 for MicroG based on Android 12.1**.
+Although this method should work on all Android versions that support Bromite and its WebView, **currently testing has only been done on LineageOS 19.1 for MicroG based on Android 12.1**.
 
 ## Prerequisites
 
-* Treble-enabled ROM ([How to check?](https://github.com/phhusson/treble_experimentations/wiki/Frequently-Asked-Questions-%28FAQ%29#how-can-i-check-if-my-device-is-treble-enabled))
+* Treble-enabled ROM (`adb shell getprop ro.treble.enabled`, [more details](https://github.com/phhusson/treble_experimentations/wiki/Frequently-Asked-Questions-%28FAQ%29#how-can-i-check-if-my-device-is-treble-enabled))
 * Custom recovery **(preferred)**, support for rooted debugging (enabled via *Settings > Developer options > Rooted debugging*) or root access
 
-## Installation
+## Installation as a Magisk module
+
+Our `BromiteSystemWebViewMagisk.zip` can be installed directly as a Magisk module. Simply `adb push` it to your device, then install it via the normal Magisk UI.
+
+This is the preferred option if you have Magisk, as some ROMs (in particular MicroG ROMs) do not have enough space to install addons. (Details in #5 and lineageos4microg/docker-lineage-cicd#358)
```

Comment:
Also mention that this is included for convenience only and only if you have Magisk already installed, as I personally do not endorse Magisk.